tests/test_token_indexer.py

```diff
@@ -1,92 +1,249 @@
 """Tests for the PretrainedBertIndexer module."""
-import unittest
-import torch
 
 from allennlp.data.tokenizers import WordTokenizer
```

Comment:
The latest allenNLP no longer has the `WordTokenizer`. To ensure updating allenNLP will not cause major break, could you add `self.tokens` manually to replace the `WordTokeizer` and `BertBasicWordSplitter` below? 

I have done it here:https://github.com/EducationalTestingService/gector/commit/d3415a1f3f8910c9ca70de9dbac121cf36e7190c#diff-585ad4e57fe86a9530e3d0d2d6ca3c3694f68c3d3c11d2adc5cfceb65889dc86R28

Reply:
By latest version, do you mean v2.10? Or is this for v0.9/v0.84?

Reply:
By latest version, I mean v2.10. v0.84 - 0.9 has the `WordTokenizer`, That is gone in the v2.10.

Reply:
Ohh, I just got it. I thought you were asking me to update it to be compatible with the latest version of allennlp, but this is just so that test outputs don't change when we have to remove `WordTokenizer` in the next PR, right?

Will do!